### DISCUSSION TITLE
Discuss: Should `GrantRound`s allow deployer to set a cap on match funding

### DISCUSSION THREAD

**Discussion by mds1** [dcgtc/dgrants#52] [Architecture & Design]
For example, do we want something like this:
1. Call `addMatchingFunds` to add matching funds to a round
2. Check if the amount being added results in total matching funds being above max allowed matching funds
3. If so, revert

TBD what to do with tokens transferred directly to the round

**Comment by thelostone-mc** [comment]
I know we discussed this the other day but figuring how to handle this extra seems to be adding extra scope.
I'd say maybe we get to this towards the end ?
